osbs/conf.py

```diff
@@ -336,14 +336,36 @@ def get_source_secret(self):
         """
         return self.get_pulp_secret()
 
-    def get_pdc_secret(self):
-        return self._get_value("pdc_secret", self.conf_section, "pdc_secret")
+    def get_smtp_host(self):
+        return self._get_value("smtp_host", self.conf_section, "smtp_host")
 
-    def get_pdc_url(self):
-        return self._get_value("pdc_url", self.conf_section, "pdc_url")
+    def get_smtp_from(self):
+        return self._get_value("smtp_from", self.conf_section, "smtp_from")
 
-    def get_smtp_uri(self):
-        return self._get_value("smtp_uri", self.conf_section, "smtp_uri")
+    def get_smtp_additional_addresses(self):
+        value = self._get_value("smtp_additional_addresses", self.conf_section,
+                                "smtp_additional_addresses")
+
+        if value:
+            return value.split(',')
```

Comment:
`get_registry_uris`, `get_registry_secrets`, and `get_registry_api_versions` use `','`
`get_token_secrets` uses whitespace
I think introducing a third one is just going to add to the confusion.